apps/omg_watcher/lib/exit_processor.ex

```diff
@@ -294,6 +305,27 @@ defmodule OMG.Watcher.ExitProcessor do
     {:reply, canonicity_result, state}
   end
 
+  def handle_call({:create_challenge, Utxo.position(blknum, txindex, oindex) = exiting_utxo_pos}, _from, state) do
+    with spending_blknum_response <- exiting_utxo_pos |> Utxo.Position.to_db_key() |> OMG.DB.spent_blknum(),
+         %{txhash: txhash} <- OMG.Watcher.DB.Transaction.get_by_position(blknum, txindex),
+         {:ok, exit_id} <- OMG.Eth.RootChain.get_standard_exit_id(txhash, oindex),
+         {:ok, raw_spending_proof, exit_info} <-
+           Core.get_challange_data(spending_blknum_response, exiting_utxo_pos, state) do
+      spending_proof =
+        case raw_spending_proof do
+          blknum when is_number(blknum) ->
```

Comment:
why do you check if blknum is a number?

Reply:
Because `raw_spending_proof` can be a  number or signed_tx

Reply:
My mistake, a different name would be useful.
Previous variable`Utxo.position(blknum ...`  mixes with this